Dockerfile

```diff
@@ -11,8 +12,10 @@ ARG     GIT_TAG
 ENV     VERGEN_GIT_SHA=${COMMIT_SHA} VERGEN_GIT_COMMIT_TIMESTAMP=${COMMIT_DATE} VERGEN_GIT_SEMVER_LIGHTWEIGHT=${GIT_TAG}
 ENV     RUSTFLAGS="-C target-feature=-crt-static"
 
-COPY    . .
-RUN     set -eux; \
+COPY    --link . .
```

Comment:
We can expect the cache is utilized better because `COPY` is done through `scratch` container.
See https://github.com/moby/buildkit/blob/dockerfile/1.4.3/frontend/dockerfile/docs/syntax.md#linked-copies-copy---link-add---link